app/helpers/simple_server_env_helper.rb

```diff
@@ -0,0 +1,26 @@
+module SimpleServerEnvHelper
+  SIMPLE_SERVER_ENV = "SIMPLE_SERVER_ENV".freeze
+
+  CUSTOMIZED_ENVS = %w[qa staging sandbox production].freeze
```

Comment:
We should include `development` in the list.

Reply:
Added `development` to `CUSTOMIZED_ENVS`.